chia/wallet/coin_selection.py

```diff
@@ -56,6 +56,11 @@ async def select_coins(
             f"Transaction for {amount} is greater than spendable balance of {sum_spendable_coins}. "
             "There may be other transactions pending or our minimum coin amount is too high."
         )
+    if amount == 0 and sum_spendable_coins == 0:
+        raise ValueError(
+            "No coins available to spend, you can not create a coin with an amount of 0,"
+            " without already having coins."
+        )

```

Comment:
I see the point of this raise, but is there any scenario where we would want to call `select_coins` and have it return an empty list?  It wouldn't hurt to have this in here, but I don't see the point.

Reply:
No, as we assert it later, however i want a clean and easy to understand error